File: 02. Back-end/src/MyEvents.API/Domain/Entity/Event/Model/Event.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MyEvents.API.Domain.Entity.Model
{
    public class Event
    {
        [Key]
        public uint IdEvent { get; set; }
        public string Local { get; set; }
        public string EventDate { get; set; }
        public string Theme { get; set; }
        public uint NumberOfParticipants { get; set; }
        public string ImageUrl { get; set; }
        public uint Telephone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Batch> Batch { get; set; }
        public IEnumerable<SocialNetwork> SocialNetwork { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvent { get; set; }
    }
}

File: 01. Front-end/MyEvents/src/app/common/model/event.ts
import { Speaker } from './speaker';
import { Batch } from "./batch";
import { SocialNetwork } from "./social-network";

export interface Event {
    idEvent: number;  
    local: string;
    eventDate?: Date;
    theme: string;
    numberOfParticipants: number;
    batch: Batch[]; 
    socialNetwork: SocialNetwork[]; 
    imageUrl: string;
    telephone: string;
    email: string;
    speakerEvent: Speaker[];
}